homeassistant/components/fastdotcom/config_flow.py

```diff
@@ -0,0 +1,35 @@
+"""Config flow for Fast.com integration."""
+from __future__ import annotations
+
+from typing import Any
+
+from homeassistant.config_entries import ConfigFlow
+from homeassistant.data_entry_flow import FlowResult
+
+from .const import DEFAULT_NAME, DOMAIN
+
+
+class FastdotcomConfigFlow(ConfigFlow, domain=DOMAIN):
+    """Handle a config flow for Fast.com."""
+
+    VERSION = 1
+
+    async def async_step_user(
+        self, user_input: dict[str, Any] | None = None
+    ) -> FlowResult:
+        """Handle the initial step."""
+        if self._async_current_entries():
+            return self.async_abort(reason="single_instance_allowed")
+
+        if user_input is not None:
+            await self.async_set_unique_id(DOMAIN)
```

Comment:
No need to set unique id

Reply:
Done.

Reply:
Setting unique id still here?

Reply:
It should have been removed in this commit: https://github.com/home-assistant/core/pull/98686/commits/3f918fe031d71da1a8df135822b166c47a04fd75